funding/manager.go

```diff
@@ -1582,7 +1585,8 @@ func (f *Manager) fundeeProcessOpenChannel(peer lnpeer.Peer,
 	case public && scid:
 		err = fmt.Errorf("option-scid-alias chantype for public " +
 			"channel")
-		log.Error(err)
+		log.Errorf("Cancelling funding flow for public channel %v "+
+			"with short channel id: %v", cid, err)
```

Comment:
nit: all channels have a short channel ID, the `scid` variable here stands for "SCID alias", so an additional, fake SCID.

Reply:
yeah this needs to be fixed otherwise this logging can be very confusing.

Reply:
I cleared up the confusion in this message and squashed the commits together.